libc/src/time/time_utils.h

```diff
@@ -156,6 +156,13 @@ LIBC_INLINE struct tm *gmtime_internal(const time_t *timer, struct tm *result) {
   return result;
 }
 
+// TODO: localtime is not yet implemented and a temporary solution is to
+//       use gmtime, https://github.com/llvm/llvm-project/issues/107597
+LIBC_INLINE struct tm *localtime(const time_t *timer) {
+  struct tm *result = {0};
```

Comment:
this should be `static struct tm result;` and then you should pass `&result` below. Since a pointer to this memory will be returned to the caller we need to make sure it's static.

Reply:
Sure, I have now made `result` `static`. I just noticed that `tm` gets intialized in `update_from_seconds`, so there is no need to initialize `result` in this function.